### PR TITLE
Added configuration for NLP and Translation APIs and basic testing of features

### PR DESCRIPTION
**Pull request by ananda1066**
Added some tests for configuration of two APIs in NLP class and tested at top of activity feed. 

NOTE: To run on your computer, make sure to add an environment variable called GOOGLE_APPLICATION_CREDENTIALS and set it to the path to get to the json file. It should look something like "C:\Users\username\Downloads\[FILE_NAME].json". Also, make sure to test it using the-java-llamas-imhere app engine, not your local one. 

And when you first run it, it will install a bunch of Google Cloud packages.

### CODE REVIEW

File: IMhere-5ef7da2d46d1.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
```
Comment (srjoglekar246) [inline IMhere-5ef7da2d46d1.json]: As suggested offline, lets explore KMS.

File: src/main/java/codeu/natural_language/NaturalLanguageProcessing.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import com.google.cloud.translate.Translation;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.Document.Type;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.lang.Exception;
+
+/** class processing messages for tone and translation */
+public class NaturalLanguageProcessing {
+  public static float testConfigurationNaturalLanguage() throws Exception{
```
Comment (srjoglekar246) [inline src/main/java/codeu/natural_language/NaturalLanguageProcessing.java]: All-encompassing `Exception` references are dangerous. Referring to explicit subclasses (the actual Exception-types that could occur) makes behavior clearer.

Also, either:
1. Have a `try` with the corresponding `catch` that handles relevant exceptions (maybe not all of them).
OR
2. Don't have `try` and instead specify all Exception subclasses to the method declaration.

File: src/main/java/codeu/natural_language/NaturalLanguageProcessing.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.Document.Type;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.lang.Exception;
+
+/** class processing messages for tone and translation */
+public class NaturalLanguageProcessing {
```
Comment (srjoglekar246) [inline src/main/java/codeu/natural_language/NaturalLanguageProcessing.java]: Add tests class. Might be worth taking text inputs to the two functions. Then, in your tests, input different words/n-grams and check if the corresponding scores match expectation. (Also border cases such as empty strings)

**Comment by ananda1066** [comment]
I've attempted to use Google KMS for the past four days but I'm not able to decrypt the JSON during the build because of gcloud configurations that are kind of complicated. I plan to come back to it after finishing my main functions, but the best I can do right now is encrypt the JSON and provide the command to decrypt it in the README or somewhere else instead if that works temporarily.

**Comment by ananda1066** [comment]
Tests for NLP take about 30 seconds to run but I left them since they only need to be run once to ensure configurations are correct. Also, I deleted the json file because it can be created automatically now - I stored the PRIVATE_KEY_ID and PRIVATE_KEY in environment variables (idk where to share those - maybe the chat?) and add them to a string that is written to a file when running the server. The function (in the JsonConfiguration file) is either called through the tests in NLP or if you skip the tests, uncomment the line in the ServerStartupListener class that calls it (there are comments pointing to it).

**Pull request merged**